### ISSUE TITLE
NKF-AEC training

### ISSUE THREAD

**Issue by Yzichen** [ayh2bxa/realtime_nkf_aec#1]
Thank you for such a great job. 
However, when I train NKF-AEC, the loss always doesn't converge and I never received a reply from the author. Can you please tell me if you were able to train it ? If so, can I add your wechat for advice?

**Comment by ayh2bxa** [comment]
I didn't train anything. The weights are already pre-trained from the original python repo, and I just copied them into the code.
